docs/modules/reference/pages/telemetryd/protocols/bmp/persisting-adapter.adoc

```diff
@@ -4,7 +4,9 @@
 
 The BMP Persisting adapter persists OpenBMP messages received without the need for any additional OpenBMP components.
 It handles BMP messages the xref:telemetryd/protocols/bmp/parser.adoc[BMP Parser] receives and parses, and creates OpenBMP-compatible messages, which are then
-persisted in OpenNMS Postgres DB.
+persisted in the OpenNMS PostgreSQL DB.
+
+//are there reasons why users wouldn't want to run this?
```

Comment:
They should have right setup for Postgres with TimeScale  and also for more throughput they should use Sentinels.

BMP Persisting Adapter can produce huge volumes of data. Without right setup, this can cause issues with other OpenNMS functionality